_includes/blogs.html

```diff
@@ -1,4 +1,9 @@
 <!-- blog -->
+
+<head>
+  <meta name="description" content="Want to know more about the Julia blog? Click here to read the latest developments in the Julia Programming Language."/>
```

Comment:
"read the latest developments in the Julia Programming Language" should maybe be "read the latest developments about the Julia Programming Language"